src/app/settings/routes.lazy.ts

```diff
@@ -16,5 +17,10 @@ export const states: ReactStateDeclaration[] = [
     name: 'gdrive-revisions',
     component: GDriveRevisions,
     url: '/settings/gdrive-revisions'
+  },
+  {
+    name: 'diagnostics',
+    url: '/diagnostics',
```

Comment:
If you make this `/settings/diagnostics` it'll lazy load correctly. Or you can have a separate lazy state for it.